app/assets/javascripts/components/amounttable.js

```diff
@@ -0,0 +1,313 @@
+import {toCSVString, downloadData} from "../utils.js";
+
+
+/**
+ * @typedef {Object} AmountTableSettings
+ * @property {string}   el        A CSS selector of the target container
```

Comment:
Is this really a string? Lower in this file, `append` is called on it. Is it a jQuery object? If so, our convention is to use variable names starting with a `$`.

_[Original comment](https://github.ugent.be/unipept/unipept/pull/625#discussion_r13715) by @bmesuere on Mon Feb 19 2018 at 13:43._